DataManagement/BranchTutorial.Rmd

```diff
@@ -96,20 +97,22 @@ Let's talk about branches.
 
 A branch lets you develop code or test an idea without affecting the original project. 
 This means you can bravely develop code that works or does not work. 
-Once you are happy with the new code, or the idea is tested and working you can merge the branch you are working on with master branch, if you are working on your own project. If you are collaborating with other people you do not want to merge with the master branch. This is when pull requests come in handy (see below). 
+Once you are happy with the new code, or the idea is tested and working you can merge the branch you are working on with the master branch, if you are working on your own project. If you are collaborating with other people you do not want to merge with the master branch. This is when pull requests come in handy (see below). 
+
 If the idea did not work or you have another idea, the branch can be deleted and nobody ever needs to know about it. 
-Make a new branch and start all over again. None of this will affect the original project.
+Make a new branch and start all over again. 
+None of this will affect the original project.
 
 Let's have a look how to create, use and delete branches.
-<br/><br/>
-<br/><br/>
+
 
 ### Make new branch (RStudio)
 
 In RStudio, click on the New Branch button in the Git window. 
 The box next to it will tell you in which branch you are. 
 Master is the default and original branch.
```

Comment:
Should we change this to "Main" now - the default from next month - or change in a month?

Reply:
i was thinking the same. it's only a couple of weeks, and it won't be ready before that anyway...